src/hera/client.py

```diff
@@ -1,5 +1,5 @@
 """Holds client configurations for communicating with Argo APIs"""
-from argo.workflows.client import ApiClient as ArgoApiClient
+from argo_workflows.api_client import ApiClient as _ArgoApiClient
```

Comment:
It appears slightly inconsistent I feel. Sometimes the imports from argo are used as is ie `from argo_workflows.models import IoArgoprojWorkflowV1alpha1Artifact` and other times as above with an `as`. Is there a reason for this? Also why now make these "private" imports - I believe it takes away somewhat from the readability.

Reply:
Not all of the generated objects have the `IoArgoprojWorkflow` prefix and some objects removed versioning from their name (`V1`). That caused some clashes with the Hera objects, as is the case for `Toleration` (used to be `V1Toleration` and now migrated to `Toleration`, motivating the `import ... as ArgoToleration`). As for the private imports, I can definitely see how it affects readability and consistency. Now that you pointed it out, we should probably stick to a consistent pattern - I will remove the private imports. As for the `as` statement, I think that is required, for now, so that we do not break Hera's simple interface. For instance, I would stand against a rename of `Toleration` to `HeraToleration`. I'd rather have Hera use alias imports so the client can avoid the need to do that with `import ... HeraToleration as Toleration`.